doc/ref_model/chapters/chapter02.md

```diff
@@ -266,16 +266,44 @@ By trying to sort workloads into different categories based on the requirements
+
+### 2.4.1 Infrastructure profiles (top-level)
+
+Based on the above analysis, the following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
 - **Basic**: for Workloads that can tolerate resource over-subscription and variable latency.
-- **Network Intensive**: for Workloads that require predictable computing performance, high network throughput and low network latency.
+- **High Performance**: for Workloads that require predictable computing performance, high network throughput and low network latency.
 
 <p align="center"><img src="../figures/ch02_infra_profiles.PNG" alt="infra_profiles" title="Infrastructure Profiles" width="100%"/></p>
 <p align="center"><b>Figure 2-1:</b> Infrastructure profiles proposed based on VNFs categorisation.</p>
 
-In **[Chapter 4](./chapter04.md)** these **B (Basic)** and **N (Network intensive)** infrastructure profiles will be defined in greater detail for use by workloads:
+In **[Chapter 4](./chapter04.md)** these **B (Basic) Performance** and **H (High) Performance** infrastructure profiles will be defined in greater detail for use by workloads.
+
+Profiles partition the infrastructure: an infrastructure object (host/node) **must** have one and only one profile associated to it.
+
+### 2.4.2 Node flavours
+
+Node flavours are meant to be used as labels for infrastructure, identifying the nodes that implement special capabilities that go beyond the profile baseline. Certain flavours may be relevant only for some profiles.
+The following **node flavours** are proposed:
+
+- **High performance CPU**: for Workloads that require predictable computing performance (or higher clock speed).
+- **High performance storage**: for Workloads that require low storage latency and/or high storage IOPS.
+- **High memory**: for Workloads that require high amounts of RAM.
+- **GPU**: denotes the presence of a consumable GPU on the infrastructure
+- **High performance network (25G)**: denotes the presence of network links (to the DC network) of speed of 25 Gbps or greater.
+- **High performance network (100G)**: denotes the presence of network links (to the DC network) of speed of 100 Gbps or greater.
+- **High Latency** - Core Sites: labels a host/node as located in a central/core site, for workloads requiring geographical centralisation.
+- **Low Latency** - Edge Sites: labels a host/node as located in an edge site, for workloads requiring low latency to final users or geographical distribution.
```

Comment:
Can we remove "edge sites"? "final users"?

The following comment is for both low and high latency:
1. Latency will depend upon the location of the users
2. Based on the above, latency is not a node flavour but a workload flavour characteristic. Workloads specify their latency sensitiveness and are placed close to users -- could be edge site or even a data center if it meets latency
3. Do we need to specify the latency values "< 10 ms" etc?

Reply:
agreed, changed to node latency labels in https://github.com/cntt-n/CNTT/commit/0ffa1b8d42c44c54cb877dac77ddca4e68062c1a